mailpoet/assets/js/src/wizard/welcome_wizard_controller.tsx

```diff
@@ -60,13 +60,17 @@ function WelcomeWizardStepsController({
       });
   }
 
-  async function finishWizard() {
+  const finishWizard = useCallback(async (redirect_url = null) => {
```

Comment:
I know we discussed moving this function to its own component, but I didn't implement it because it was not clear to me how to reuse the `loading` state that is defined inside `updateSettings()` and used in the `WelcomeWizardStepsController` component.

I agree that I should do something about it, as passing this function around the components doesn't look good.

Reply:
I ended up extracting `updateSettings`, `wizard-redesign-mss-step-different-look` branch includes the proposed change, however it's better to test it together since I'm  not exactly sure how the feature should behave.

Reply:
@samnajian, maybe you forgot to push the branch? I'm not able to find it.